docs/kratos/emails-sms/05_custom-email-templates.mdx

```diff
@@ -0,0 +1,285 @@
+---
+id: custom-email-templates
+title: Customize message templates for recovery and verification emails
+sidebar_label: Email templates
+---
```

Comment:
@Benehiko I'd appreciate it if you could review this document :)

Reply:
Hey @tomekpapiernik 
I just tried out the approach with Ory Cloud and it worked fine :) 
The nested templates section needs an Ory Cloud part too right?

Reply:
Oh misread that, so "Nested templates containing remote templates" are possible only in self-hosted right now and normal nested should work fine? in that case I can do some testing.

Reply:
"Normal" as in all the logic has to be in a single template? Yes. 
Move the "nested with remote" thing to that self-hosted doc?